cloudflare/resource_cloudflare_access_group_test.go

```diff
@@ -0,0 +1,359 @@
+package cloudflare
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+var (
+	accountID   = "01a7362d577a6c3019a474fd6f485823"
+	email       = "test@example.com"
+	accessGroup cloudflare.AccessGroup
+)
+
+func TestAccCloudflareAccessGroupConfig_Basic(t *testing.T) {
+	name := "cloudflare_access_group.test"
+	resourceName := strings.Split(name, ".")[1]
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
```

Comment:
This should also include [`testAccPreCheckAccount`](https://github.com/terraform-providers/terraform-provider-cloudflare/blob/7df730622f105a6737afedd2c24453c855b32322/cloudflare/provider_test.go#L62-L66) now that we're referencing it. You'll need to update all occurrences of it for it to be checked correctly.